sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -44,24 +44,6 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     ${level-zero-loader_SOURCE_DIR}/include CACHE PATH "Path to Level Zero Headers")
 endif()
 
-add_library (LevelZeroLoader INTERFACE)
```

Comment:
comment here https://github.com/oneapi-src/unified-runtime/pull/879/files#r1334743502 about the L0 loader tag.